config/default.schema.json

```diff
@@ -331,11 +331,29 @@
 				}
 			}
 		},
+		"H5P_EDITOR": {
+			"type": "object",
+			"description": "Properties of the H5P server microservice",
+			"default": {},
+			"properties": {
```

Comment:
@Loki-Afro  Maybe I'm misunderstanding, are we talking about the entire Config?
I've added the INCOMING_REQUEST_TIMEOUT config because all other microservices have their own version of it, it is already used when starting the service.
I could remove the entire config and just use the existing timeout that is meant for the main server (INCOMING_REQUEST_TIMEOUT_API), if that's what you mean

Reply:
Removed config for H5P since it is not needed right now. Use the normal INCOMING_REQUEST_TIMEOUT_API for H5P too